src/components/range/range.ts

```diff
@@ -285,6 +285,12 @@ export class Range extends BaseInput<any> implements AfterContentInit, ControlVa
   }
 
   /** @internal */
+  _calcRatio(current: PointerCoordinates, rect: ClientRect): number {
+    let ratio = clamp(0, (current.x - rect.left) / (rect.width), 1);
+    return this._plt.isRTL ? 1 - ratio : ratio;
```

Comment:
> Typescript only has one "change", which is to flip the X coordinate, (but easier is to flip the ratio which is already normalised)

But this works on the fly. the `_calcRatio` method I added reduced a code clone that calculated ratio in 2 places.